packages/css/src/card/fullwidth-card.css

```diff
@@ -0,0 +1,128 @@
+@import url("../spacing/spacing.css");
+
+.hds-card.hds-card--full-width {
+  /* Basic sizing for fullwidth cards: */
+  --hds-fullwidthcard-min-height: 400px;
+  --hds-fullwidthcard-content-min-height: calc(
+    var(--hds-fullwidthcard-min-height) - 2 * var(--hds-spacing-20-24)
+  );
+  --hds-fullwidthcard-image-aspect-mini: 100 / 155;
+  --hds-fullwidthcard-image-maxheight-small: 420px;
+  --hds-fullwidthcard-image-maxheight-medium: 650px;
+  --hds-fullwidthcard-image-height-small: 300px;
+  --hds-fullwidthcard-image-height-medium: 600px;
+
+  max-width: inherit;
+  container-type: inline-size;
+  container-name: fullwidthcard;
+
+  * {
+    box-sizing: border-box;
+  }
+
+  .hds-card__layoutwrapper {
+    height: 100%;
+    width: 100%;
+  }
+
+  .hds-card__media {
+    overflow: hidden;
+    position: relative;
+    border-radius: var(--hds-border-radius-8);
+    max-height: var(--hds-fullwidthcard-image-maxheight-small);
+    width: 100%;
+
+    /*  Responsivity: different card layouts and content/images sizes, while still displaying as much as the image as
+     *  possible and trying to keep focus on the middle, .hds-card__media is displayed in particular ways on
+     *  particular viewport widths, but always filling the available width (which below 800px is the full width of the
+     *  card minus padding, and above 800px is only half the width of the card):
+     *
+     **/
+
+    /* Narrowest: fixed aspect ratio (portrait: height is 1.5 times the width). */
+    @container fullwidthcard (width < 300px) {
+      aspect-ratio: var(--hds-fullwidthcard-image-aspect-mini);
+      max-height: var(--hds-fullwidthcard-image-height-small);
+    }
+
+    /* Middle narrow: fixed aspect ratio (quare format), variable height */
+    @container fullwidthcard (width >= 300px) and (width < 648px) {
+      aspect-ratio: 1;
+    }
+
+    /* "Widest" of the narrow card layouts: fixed image height (but higher now) */
+    @container fullwidthcard (width >= 648px) and (width < 800px) {
+      height: var(--hds-fullwidthcard-image-height-medium);
+    }
+  }
+
+  .hds-card__body {
+    .hds-card__centerbody {
+      margin: var(--hds-spacing-32-40);
+    }
+
+    .hds-card__body-description {
+      margin-top: var(--hds-spacing-20-24);
+    }
+
+    .hds-stack {
+      gap: 8px;
+    }
```

Comment:
Targeting the stack is really hacky. Can this be solved in another way?

Reply:
We should rather then make a `<Card.BodyActions>` or `<Card.BodyActionsRow>` component that wraps `HStack` if this is common pattern. Instead of creating an implicit link to the `HStack` component